ClearKeep/App/System/PushNotificationsHandler.swift

```diff
@@ -31,23 +35,103 @@ extension PushNotificationsHandler: UNUserNotificationCenterDelegate {
 								willPresent notification: UNNotification,
 								withCompletionHandler completionHandler:
 								@escaping (UNNotificationPresentationOptions) -> Void) {
+		self.handleNotification(userInfo: notification.request.content.userInfo, completionHandler: {})
 		completionHandler([.banner, .sound, .list])
 	}
 	
 	func userNotificationCenter(_ center: UNUserNotificationCenter,
 								didReceive response: UNNotificationResponse,
 								withCompletionHandler completionHandler: @escaping () -> Void) {
 		let userInfo = response.notification.request.content.userInfo
+		print("notification payload: \(userInfo)")
 		handleNotification(userInfo: userInfo, completionHandler: completionHandler)
 	}
 	
 	func handleNotification(userInfo: [AnyHashable: Any], completionHandler: @escaping () -> Void) {
-		guard let payload = userInfo["aps"] as? NotificationPayload else {
-			completionHandler()
+		if let jsonString = userInfo["publication"] as? String,
+		   let jsonData = jsonString.data(using: .utf8) {
+			
+			if let publicationJson = try? JSONSerialization.jsonObject(with: jsonData, options: []) as? [String: Any],
+			   let deactiveAccountId = publicationJson["deactive_account_id"] as? String {
+				handleLogout(deactiveAccountId: deactiveAccountId)
+				completionHandler()
+				return
+			}
+			
+			do {
+				let publication = try JSONDecoder().decode(PublicationMessageNotification.self, from: jsonData)
+				//container.appState[\.authentication.selectedGroupId] = publication.groupId
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/App/System/PushNotificationsHandler.swift#L63"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>PushNotificationsHandler.swift:63</code></td>
    </tr>
  </tbody>
</table>